contracts/InsuranceArbitrator.sol

```diff
@@ -30,28 +30,20 @@ contract InsuranceArbitrator {
         bool claimInitiated; // Claim state preventing simultaneous claim attempts.
         string insuredEvent; // Short description of insured event.
         address insuredAddress; // Beneficiary address eligible for insurance compensation.
-        IERC20 currency; // Denomination token for insurance coverage.
         uint256 insuredAmount; // Amount of insurance coverage.
     }
 
-    // Tracks raised claims on insurance policies.
-    struct Claim {
-        bytes32 policyId; // Claimed policy identifier.
-        OptimisticOracleV2Interface optimisticOracle; // optimistic oracle instance where claims are resolved.
-    }
-
     // References all active insurance policies by policyId.
     mapping(bytes32 => InsurancePolicy) public insurancePolicies;
 
-    // Maps hash of initiated claims to their policyId and optimistic oracle implementation.
+    // Maps hash of initiated claims to their policyId.
     // This is used in callback function to potentially pay out the beneficiary.
-    mapping(bytes32 => Claim) public insuranceClaims;
+    mapping(bytes32 => bytes32) public insuranceClaims;
 
     // Oracle proposal bond set to 0.1% of claimed insurance coverage.
-    uint256 constant oracleBondPercentage = 1e15;
+    uint256 public constant oracleBondPercentage = 1e15;
```

Comment:
this might be easier to read...

```suggestion
    uint256 public constant oracleBondPercentage = 0.001e18; // Oracle proposal bond set to 0.1% of claimed insurance coverage.

```